staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go

```diff
@@ -22,6 +22,19 @@ limitations under the License.
 // leader (a.k.a. fencing). A client observes timestamps captured locally to
```

Comment:
Maybe break the first sentence of this paragraph into the previous paragraph so that discussion on the finer points of timing is cleanly separated.

Reply:
Will do.